draft-documents/satisfying-real-world-use-cases.md

```diff
@@ -436,7 +435,7 @@ Thus, the main technical needs for Marsha's case are:
 
 The key to meeting these needs is to provide an individual services across time
 and space, in a way that avoids duplication and provides for continuity of care
-(medically)--in other words, persistent correlatable claims.  An independent
+(medically)--in other words, persistent correlatable identifier - along with records associated with it.  An independent
```

Comment:
The way this is worded sounds like records will actually be released, but a persistent theme of this document is that (where possible) records themselves are never exfiltrated from their point of creation or storage. Do we have any more room to go into the nuances?